packages/common/src/interfaces/AudioInterface.ts

```diff
@@ -0,0 +1,9 @@
+export interface AudioInterface {
+  id: string
+  name?: string
+  tags?: string[]
+  duration: number
+  mp3StaticResourceId?: string
+  oggStaticResourceId?: string
+  mpegStaticResourceId?: string
```

Comment:
No reason for these to be separate static resources. An item in the audio table should point to a single static resource. That resource can be represented in a variety of formats / lods.